integrationtest/command/testdata/print_conf_flags_and_file/after/a.yaml

```diff
@@ -0,0 +1,2 @@
+a:
```

Comment:
These yaml files can be removed for these tests `-print_conf` tests. They should be able to operate without them.